references.bib

```diff
@@ -218,3 +219,10 @@ @inproceedings{kluyver2016jupyter
   volume = {87},
   address = {Amsterdam}
 }
+
+@online{stanfordhealthcare,
```

Comment:
There's something weird about how this is rendering. I see this on the rendered page:

"Stanford Health Care. 2021. “What Is Cancer?” 2021. https://stanfordhealthcare.org/medical-conditions/cancer/cancer.html."

2021 is printed twice...

Reply:
This seems to be an issue with a couple of our references so I made a new issue #243